src/components/common-components/autocomplete/autocomplete.component.tsx

```diff
@@ -20,7 +26,13 @@ interface AutocompleteOptions<T> {
  * Dropdown create by this function is always opened.
  * To close the dropdown, you should destroy this component
  */
-export function AutocompleteComponent<T>({
+export function AutocompleteComponent<
```

Comment:
Moja propozycja jest taka, żeby pozbyć się tego `autocompleteComponentu i przenieść to wszystko do `ShiftAutocompleteComponent`.
Taka propozycja dlatego, że teraz ten dropdown już nie jest generyczny i jest bardzo dostosowany do dropdownów z dyżurami w związku z czym nie widzę też sensu trzymać go w osobnym komponencie.

Reply:
Dobra, będzie zrobione.

Reply:
Ok